compiler/control/OMROptions.cpp

```diff
@@ -2315,6 +2315,10 @@ OMR::Options::jitLatePostProcess(TR::OptionSet *optionSet, void * jitConfig)
             fej9->waitOnCompiler(jitConfig);
             }
          }
+
+      // disable regDebCopyRemoval when value types are enabled
```

Comment:
Sp:  regDepCopyRemoval

Reply:
It seems to me that the text you have in the PR commit explaining why you have to disable this would be a useful comment to add in the code here.

Reply:
Good idea! Fixed.